tests/phpunit/tests/editor/disableContentEditorForNavigationPostType.php

```diff
@@ -0,0 +1,63 @@
+<?php
+/**
+ * @group editor
+ *
+ * @covers ::_disable_content_editor_for_navigation_post_type
+ */
+class Tests_Editor_DisableContentEditorForNavigationPostType extends WP_UnitTestCase {
+
+	const NAVIGATION_POST_TYPE     = 'wp_navigation';
+	const NON_NAVIGATION_POST_TYPE = 'wp_non_navigation';
+
+	public function tear_down() {
+		$this->enable_editor_support();
+		parent::tear_down();
+	}
+
+	/**
+	 * @ticket 56266
+	 */
+	public function test_it_doesnt_disable_content_editor_for_non_navigation_type_posts() {
+		$post = $this->create_non_navigation_post();
+		$this->assertTrue( $this->supports_block_editor() );
```

Comment:
All test methods in the PR containing multiple assertions need a `message` parameter added to each assertion. [[Ref]](https://make.wordpress.org/core/handbook/testing/automated-testing/writing-phpunit-tests/#using-assertions)

Reply:
Yes, we discussed error messages some time ago. I didn't notice that for some reason.
Thank you for pointing it out.
Fixed in 8da8362eb2ae2083d2d5e5e81cfa454526442005